include/cuda/std/detail/libcxx/include/algorithm

```diff
@@ -13,7 +13,7 @@
 /*
     algorithm synopsis
 
-#include <initializer_list>
+#include <::std::initializer_list>
```

Comment:
Tell me you've just run search+replace without telling me... :D

Reply:
Also fixed. >.>